src/main/java/org/spongepowered/common/registry/SpongeGameRegistry.java

```diff
@@ -397,7 +394,7 @@
     private final Map<String, CookedFish> cookedFishMappings = Maps.newHashMap();
     private final Map<String, DyeColor> dyeColorMappings = Maps.newHashMap();
     private final Map<String, Art> artMappings = Maps.newHashMap();
-    private final Map<String, EntityType> entityTypeMappings = Maps.newHashMap();
+    protected final Map<String, EntityType> entityTypeMappings = Maps.newHashMap();
```

Comment:
There is always a need to verify imo. Seems better to have a protected method and keep the map private.